framework-docs/modules/ROOT/pages/core/beans/classpath-scanning.adoc

```diff
@@ -323,6 +323,45 @@ sure that they are 'opened' (that is, that they use an `opens` declaration inste
 `exports` declaration in your `module-info` descriptor).
 ====
 
+==== Property placeholders and Ant-style patterns
```

Comment:
Please note that this breaks the build.

```
[16:49:49.179] WARN (asciidoctor): section title out of sequence: expected level 2, got level 3
    file: /XXX/spring-framework/framework-docs/modules/ROOT/pages/core/beans/classpath-scanning.adoc:335
```

However, I have addressed this locally, thus there is no need to update this PR.